kubernetes/README.md

```diff
@@ -4,70 +4,111 @@ This example explains how to deploy the 1Passwrd SCIM bridge on Kubernetes runni
+Log in to your 1Password account [using this link](https://start.1password.com/settings/provisioning/setup). It will take you to the setup page for the SCIM bridge. Follow the instructions there.
+
+*IMPORTANT*: When the setup asks you for an email address for the new Provision Manager user it creates for you automatically, use a dedicated email address (for example: `op-provision-manager@example.com`) to handle this account. It is _not advised_ to use any personal email address!
+
+*IMPORTANT*: DO NOT create the Provision User manually. Let the setup process create the Provision Manager user for you **automatically.**
+
+Once complete, you'll be provided with two separate secrets:
+
+* a `scimsession` file 
+* a bearer token
+
+The `scimsession` file contains the credentials for the new Provision Manager user the setup process automatically created for you. This user will create, confirm, and suspend users, and create and manage access to groups.  
 
-Follow the on-screen instructions which will guide you through the following steps:
+*IMPORTANT*: Do not share these secrets! The bearer token must be provided to your Identity Provider, but beyond that it should be kept safe and not shared with anyone. Especially important is the `scimsession` file - do not share that with ANYONE, except the SCIM bridge.
+
+To reiterate: you'll need to share the bearer token with your Identity Provider, but it's important to **never share it with anyone else**. And never share your `scimsession` file with **anyone at all**.
+
+## Create your `scimsession` Kubernetes secret
+
+Next, we must create a Kubernetes secret containing the scimsession file. Using `kubectl`, we can read the scimsession file and create the secret in one command:
+
+```
+kubectl create secret generic scimsession --from-file=./scimsession
+```
+
+Make sure to pass the filepath of the scimsession file that you downloaded.  The above command will look for the file in  this folder (the `/kubernetes/` folder) of the repository.
 
-* Create a Provision Managers group
-* Create and confirm a Provision Manager user
-* Generate your SCIM bridge credentials
 
-You can then download the `scimsession` file and save your bearer token.  The `scimsession` file contains the credentials for the new Provision Manager user.  This user will create, confirm, and suspend users, and create and manage access to groups.  You should use an email address that is unique.
+## Configuring the SCIM bridge
+
+You'll want to edit the `op-scim-deployment.yaml` file and change the variable `{YOUR-DOMAIN-HERE}` you've decided on in _Creating your DNS record_. This allows LetsEncrypt to issue your deployment an SSL certificate necessary for encrypted traffic. As discussed in that section, you'll need to have the IP address of the container, so we'll continue with deployment, and then finish configuring the DNS.
```

Comment:
How about we create an anchor? I'm assuming `#Creating Your DNS Record` won't work, but a link would `[Creating your DNS Record](link with anchor)`